src/braket/experimental/autoqasm/operators/return_statements.py

```diff
@@ -14,7 +14,7 @@
 
 """Operations to override return statement behavior."""
 
-from typing import Any
+from typing import Any, Iterable
```

Comment:
[nit] could use `collections.abc.Iterable`, technically` typing.Iterable` is deprecated.